### DISCUSSION TITLE
On android, v-model only updates input value when it loses focus

### DISCUSSION THREAD

**Discussion by descodifica** [quasarframework/quasar#10755] [General - Components / Directives / etc]
I have a form field with v-model, the expectation is that the variable's value is updated as the user updates.
On the computer/browser it works perfectly, but in the android application it only updates the variable value when the field loses focus.

This even gets in the way of validation, since it only releases the button when the field is valid, and it will only be valid when updating the variable...

To make sure it's not my logic or package with conflict, I created a new quasar project and just added the input with the v-model, the problem persisted.

The project link "blank" and still with error is below for those who want to do a look and/or their own test.
In short... how can I make the v-model variable value update as I type?

https://github.com/descodifica/bug-quasar-vmodel?fbclid=IwAR2wFllbFvKnkE0sFDGFR8r73x6Mf5ckTVyhqRZa7on0aMk_kmkrl3Qy3AE

**Comment by pdanpdan** [comment]
Sorry, I think it would help if you start by reading vue docs. Your value is not reactive.
